pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -391,12 +391,12 @@ svg {
 // intended mainly for PF PageSection elements (pf-c-page__main-section).
 // It's similar to adding padding={{ default: 'noPadding', sm: 'padding' }},
 // except this only affects the sides, not the top and bottom.
-@media screen and (max-width: $pf-global--breakpoint--sm) {
-  .pf-c-page__main > section.pf-c-page__main-section:not(.pf-m-padding) {
-    padding-left: 0;
-    padding-right: 0;
-  }
-}
+// @media screen and (max-width: $pf-global--breakpoint--sm) {
+//   .pf-c-page__main > section.pf-c-page__main-section:not(.pf-m-padding) {
+//     padding-left: 0;
+//     padding-right: 0;
+//   }
+// }
```

Comment:
This was done deliberately in commit 3d876b610105fb, i.e. it's not an accidental bug. Let's please first discuss this in https://github.com/cockpit-project/cockpit/issues/18230#issuecomment-1457671305 .

@MahmoudHamdy02 : In the meantime: Please don't comment out things, just delete them. Also, commit messages should say the "where" (in this case, start Subject with "patternfly:") and the "why?", i.e. explain the reason for a change. It doesn't need to restate the "what". As the "why" isn't obvious at all here, I suggest to just remove the commit for now.